package.json

```diff
@@ -40,7 +40,8 @@
   },
   "dependencies": {
     "jwt-decode": "^3.1.2",
-    "mitt": "^2.1.0"
+    "mitt": "^2.1.0",
+    "tslib": "^2.6.2"
```

Comment:
I think this is actually a dev dependency

Reply:
It is - fixed